### ISSUE TITLE
Change the correlations of trial and learn separations to only appropriate pulses

### ISSUE THREAD

**Issue by hejtmy** [hejtmy/hcenat-fmri#39]
The correlation ot moving-learn, pointing-trial etc. should be only of those pulses whcih cover the events. This might be introducing some error and would need to be corrected with bootstrapping, but the issue is that the hrf convolution is missing for non-trial movement or pointing. Therefore if a component correlates with movement generally but correlates MORE with trial movement, we would NOT see the correlation in this way. 

The solution might be to preselect strong components and then compare is there are any differences in the strenght of the correlation for trial vs learning events

So basically for those trial and leran, correlate only given pulses, not the entire set.
